packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -254,13 +307,14 @@ export class TokenRatesController extends StaticIntervalPollingControllerV1<
       }
     });
 
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
     onNetworkStateChange(async ({ providerConfig }) => {
       const { chainId, ticker } = providerConfig;
       if (
         this.config.chainId !== chainId ||
         this.config.nativeCurrency !== ticker
       ) {
-        this.update({ contractExchangeRates: {} });
+        this.update({ marketData: {} });
```

Comment:
if the default state updates we could miss having to update it here.,

```suggestion
        this.update(this.defaultState);
```